### PR TITLE
feat: (home) Replace lottery card with prediction promotion card

### PR DESCRIPTION
**Pull request by memoyil**
To review:

https://deploy-preview-1533--pancakeswap-dev.netlify.app/

Also brings back WinCard and naming change on that

### CODE REVIEW

File: src/state/predictions/helpers.ts [hunk 1]
```diff
@@ -228,6 +236,25 @@ export const getMarketData = async (): Promise<{
...
 
+export const getStats = async (): Promise<{
+  stats: Stats
+}> => {
+  const response = (await request(
+    GRAPH_API_PREDICTION,
+    gql`
+      query getMarketData {
+        market(id: 1) {
+          totalBNB
```
Comment (ChefKai) [inline src/state/predictions/helpers.ts:247]: In order to have a more accurate number to display, we should add `totalBNBTreasury` to the query, and subtract it from `totalBNB` to account for the treasury fee (used for the burn) + the `House` rounds

Comment (memoyil) [reply src/state/predictions/helpers.ts:242]: I added totalBNBTreasury to the calculation, what are the House rounds? (failed ones?)

Comment (memoyil) [reply src/state/predictions/helpers.ts:253]: @ChefKai Done, Can you review it again?

File: src/state/predictions/helpers.ts [hunk 2]
```diff
@@ -228,6 +244,25 @@ export const getMarketData = async (): Promise<{
...
+    GRAPH_API_PREDICTION,
+    gql`
+      query getTotalWonData {
+        market(id: 1) {
+          totalBNB
+          totalBNBTreasury
+        }
+        rounds(where: { position: ${BetPosition.HOUSE} }) {
+          totalAmount
+        }
```
Comment (hachiojidev) [inline src/state/predictions/helpers.ts:258]: Pass in the variables like this with the second argument. You can check `getRound` as an example.

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/state/predictions/helpers.ts (hunk 2 above)
Comment (memoyil) [reply src/state/predictions/helpers.ts]: Done

**Review by hachiojidev — APPROVED** [review]

**Review by hachiojidev — CHANGES_REQUESTED** [review]
@memoyil After looking at the preview can you do 2 things:

1. Make sure the skeleton is the same size as the final number size so it doesn't jump

2. Use the count up component.

**Comment by memoyil** [comment]
@hachiojidev Done

**Review by hachiojidev — APPROVED** [review]
Looks good.

**Pull request merged**